types/types_test.go

```diff
@@ -93,3 +94,43 @@ func TestNewHTTPCodeRanges(t *testing.T) {
 		})
 	}
 }
+
+func TestHTTPCodeRanges_Contains(t *testing.T) {
+	testCases := []struct {
+		strBlocks  []string
+		statusCode int
+		contains   bool
+	}{
+		{
+			strBlocks:  []string{"200-299"},
+			statusCode: 200,
+			contains:   true,
+		},
+		{
+			strBlocks:  []string{"200"},
+			statusCode: 200,
+			contains:   true,
+		},
+		{
+			strBlocks:  []string{"201"},
+			statusCode: 200,
+			contains:   false,
+		},
+		{
+			strBlocks:  []string{"200-299", "500-599"},
+			statusCode: 400,
+			contains:   false,
+		},
+	}
+
+	for _, test := range testCases {
+		test := test
+		testName := fmt.Sprintf("%q contains %d", test.strBlocks, test.statusCode)
+		t.Run(testName, func(t *testing.T) {
+			httpCodeRanges, err := NewHTTPCodeRanges(test.strBlocks)
```

Comment:
Could you please run these test cases in parallel

Reply:
Good point, fixed.